articles/communication-services/quickstarts/email/includes/send-email-js.md

```diff
@@ -140,11 +129,10 @@ main();
 ```
 ## Getting MessageId to track email delivery
 
-To track the status of email delivery, you need to get the MessageId back from response and track the status. If there's no MessageId retry the request.
+To track the status of email delivery, you need to get the MessageId back from response and track the status. If there's no MessageId, retry the request.
 
 ```javascript
- // check mail status, wait for 5 seconds, check for 60 seconds.
-  const messageId = response._response.parsedHeaders.xMsRequestId;
+  const messageId = response.messageId;
   if (messageId === null) {
     console.log("Message Id not found.");
     return;
```

Comment:
After the return, I'd suggest adding code that logs the messageId just to give the user some quick feedback after they make the call. This way they can also test the getStatus call separately: 

`console.log("Response received. messageId: " + response.messageId);`